doc/make/msys2.md

```diff
@@ -22,12 +22,44 @@ Here are the commands to install all dependencies needed to compile Lean on your
 pacman -S make mingw-w64-x86_64-cmake mingw-w64-x86_64-ccache mingw-w64-x86_64-gcc git
 ```
 
+Then make sure the following is included in your PATH:
+
+```bash
+export PATH=$PATH:/mingw64/bin
+```
+
+You should now be able to run these commands:
+
+```bash
+gcc --version
+cmake --version
+```
+
 Then follow the [generic build instructions](index.md) in the MSYS2 MinGW shell, using
 `cmake ../.. -G "Unix Makefiles"` instead of `cmake ../..`. This ensures that cmake will call `sh` instead
 of `cmd.exe` for script tasks.
 
 ## Install lean
 
 You can use the `install` ninja/make target to install Lean into, by default,
-`C:\\User Programs (x86)\\LEAN`. To change this, add `-DCMAKE_INSTALL_PREFIX=path/you/want`
-to your cmake invocation.
+`./build/release/stage1/msys64/lean/`. To change this, add `-DCMAKE_INSTALL_PREFIX=path/you/want`
+to your initial cmake invocation.
```

Comment:
If I try `make install`, I get
```
Install the project...
-- Install configuration: "Release"
-- Installing: C:/Program Files (x86)/LEAN/include/lean
CMake Error at cmake_install.cmake:41 (file):
  file INSTALL cannot make directory "C:/Program Files
  (x86)/LEAN/include/lean": No such file or directory.
```
Not sure if that is a permission problem.

Reply:
Probably is given that "c:/Program Files" is usually not writable by non-admin users...